source/sdk/Memory.ooc

```diff
@@ -2,17 +2,8 @@ include string // Memory routines in C are actually in string.h
 version(windows) { include malloc }
 else { include alloca }
 
-// GC_MALLOC zeroes the memory, so in the non-gc version, we prefer to use calloc
-// to the expense of some performance. Don't use malloc instead - existing ooc code
-// is written assuming that gc_malloc zeroes the memory.
-gc_malloc: func (size: SizeT) -> Pointer { gc_calloc(1, size) }
```

Comment:
We would need some kind of global flag that is set to true when entering the inner functions so that recursive calls from built in allocation is not debug printing again from the string operations. There might be other things preventing the use of println. It would be nice to have 2 debug functions where any code can be used to track memory errors. For classes, it would be very useful to get the class name and count of each leak.